app/models/case_closure/offender_complaint_appeal_outcome.rb

```diff
@@ -0,0 +1,45 @@
+# == Schema Information
+#
+# Table name: case_closure_metadata
+#
+#  id                      :integer          not null, primary key
+#  type                    :string
+#  subtype                 :string
+#  name                    :string
+#  abbreviation            :string
+#  sequence_id             :integer
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#  requires_refusal_reason :boolean          default(FALSE)
+#  requires_exemption      :boolean          default(FALSE)
+#  active                  :boolean          default(TRUE)
+#  omit_for_part_refused   :boolean          default(FALSE)
```

Comment:
Are these comments correct for this class? Just wondering if these are left behind from reusing an existing file :)

Reply:
The schema for this table case_closure_metadata:  are the same cross all those classes which make sense,   although some flags in this table may only be relevant some of them.   I am happy to remove them, just feel it is redundant or misleading :)

Reply:
but the good things to place them in metadata table rather than enum type, is to have active flag and other possible flags

Reply:
Oh, I see! Thanks for the explanation 👍